LondonBritishLibrary/orient/BLorient548.xml

```diff
@@ -31,89 +32,255 @@ schematypens="http://relaxng.org/ns/structure/1.0"
-                            
-                        </altIdentifier>
-                        
+                    </altIdentifier>
                     </msIdentifier>
-                    <msContents>
-                        <summary/>
-                        
-                        
+
+                                        <physDesc>
+                                            <objectDesc form="Codex">
+                                                <supportDesc>
+                                                    <support>
+                                                        <material key="parchment"/>
+
+                                                    </support>
+
+                                                    <extent>
+                                                        <measure unit="leaf">129</measure>
+                                                        <dimensions type="outer" unit="in">
+                                                            <height>8.875</height>
+                                                            <width>6.75</width>
+                                                        </dimensions>
+                                                    </extent>
+                                                    <condition key="deficient"><desc>Leaves torn out after <locus target="#f.65">f. 65</locus>
+                                                    and <locus target="#69">f. 69</locus></desc>.
+                                                  </condition>
+                                                </supportDesc>
+
+                                                <layoutDesc>
+                                                    <layout columns="2" writtenLines="21"/>
+                                                </layoutDesc>
+                                            </objectDesc>
+
+                                            <handDesc>
+
+                                                <handNote script="Ethiopic" xml:id="h1">
+                                                    <desc>The manuscript is written in a good hand, of the <date>19th century</date>,imperfect at the end.
+                                                    </desc>
+
+                                                    <date notBefore="1800" notAfter="1900"/>
+
+                                                </handNote>
+
+                                            </handDesc>
+                                            <decoDesc>
+                                              <decoNote type="ornamentation" xml:id="d1">
+                                              <locus target="#7r #31r"/>
+                                              <desc>Few attempts at ornament.</desc>
+                                              </decoNote>
+                                              </decoDesc>
+                                            <additions>
+
```

Comment:
Condition, layout, handnote and deco should be moved to the corresponding part (part 1 I guess).